packages/pose-core/src/factories/setter.ts

```diff
@@ -70,7 +70,7 @@ const startChildAnimations = <V, A, C, P>(
       ? (i: number) => i * stagger
       : (i: number) => maxStaggerDuration - i * stagger;
 
-  Array.from(children).forEach((child, i) => {
+  Array.from(children.values()).forEach((child, i) => {
```

Comment:
Maybe just skip `Array.from` altogether? `Set` just has a `forEach` method

Reply:
Correct, but the `Set.prototype.forEach` doesn't come with an index parameter which is used to create a staggered delay:

https://github.com/Popmotion/popmotion/blob/11009eeb33d7d7a98bc5a40b068bb65a2fd2727e/packages/pose-core/src/factories/setter.ts#L73-L80

Reply:
Right, havent noticed this by looking quickly at this. This has caught my eye as potential problem - https://github.com/Popmotion/popmotion/issues/597 , but for different reasons, so it certainly shouldnt block this PR.